### PR TITLE
fix(ci): Clippy Beta

### PR DESCRIPTION
**Pull request by hanabi1224**
## Description

Fixes CI failure in Clippy (Beta)

https://github.com/libp2p/rust-libp2p/actions/runs/12055058396/job/33614543029

## Change checklist

- [x] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] A changelog entry has been made in the appropriate crates

### CODE REVIEW

**Comment by guillaumemichel** [comment]
Thanks @hanabi1224! Could you also update the CHANGELOGs  of crates where required?

**Comment by hanabi1224** [comment]
> Thanks @hanabi1224! Could you also update the CHANGELOGs of crates where required?

@guillaumemichel Done.

File: transports/noise/CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,8 @@
+## unreleased
```
Comment (guillaumemichel) [inline transports/noise/CHANGELOG.md]:
```suggestion
## 0.45.1
```

File: transports/webtransport-websys/CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,8 @@
+## unreleased
```
Comment (guillaumemichel) [inline transports/webtransport-websys/CHANGELOG.md]:
```suggestion
## 0.4.1
```

**Review by guillaumemichel — APPROVED** [review]
Thanks @hanabi1224!

**Pull request merged**